clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientConfiguration.java

```diff
@@ -27,6 +27,12 @@ public interface ZeebeClientConfiguration {
    */
   String getGatewayAddress();
 
+  /**
+   * @see ZeebeClientBuilder#defaultTenantId(String)
+   * @return
```

Comment:
🤡 What do we return?

Reply:
Woops, I'll remove that to keep in line with the other Javadocs in this class.